packages/command-handler-svc/src/handler.ts

```diff
@@ -208,15 +218,37 @@ export class SettlementsCommandHandler{
 					}
 				}
 
-			}catch(err: unknown){
+			} catch (err) {
 				this._logger.error(err, `TransfersCommandHandler - processing command - ${message?.msgName}:${message?.msgKey}:${message?.msgId} - Error: ${(err as Error)?.message?.toString()}`);
-			}finally {
+			} finally {
 				resolve();
 			}
 		});
 	}
 
 	private async _getServiceSecContext():Promise<CallSecurityContext>{
+		if (this._barebone) {
```

Comment:
We don't need to check security context in the aggregate functions called by the command handler (as they are all system calls).
For now, we should only check for privileges in the API (create matrix, lock, etc..)